spec/import_spec.rb

```diff
@@ -2,6 +2,28 @@
 
 describe 'import', type: :feature do
 
+  shared_examples 'successful inserts' do |encoding, csv_file_name|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.